src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-inline: 50px;
+  align-items: center;
+  position: relative;
+
+  &__logo {
+    height: 40px;
+    margin: auto 0;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    align-items: center;
+  }
+
+  &__link {
+    display: flex;
+    color: $nav-text-color;
+    text-decoration: none;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 60px;
+    align-items: center;
+    position: relative;
+
+    &.is-active {
+      color: $nav-text-hover-color;
```

Comment:
The variable `$nav-text-hover-color` is used here, but it's not defined within this file. Ensure that it's defined in a parent file or imported correctly.